inst/silverstein/DESCRIPTION

```diff
@@ -0,0 +1,14 @@
+Package: silverstein
```

Comment:
👏 Love this!

Reply:
I tried to play along.  Glad you appreciate it. 😄